### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Dinhduy01**
## Related Tickets
- [#Task xxxxx: Content](https://edu-redmine.sun-asterisk.vn/issues/xxxx)
## WHAT
- Create a new Android app with a main Activity and a second Activity.
## Evidence (Screenshot or Video)
image
image

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/sun/android/MainActivity.kt [hunk 1]
```diff
@@ -1,11 +1,43 @@
...
 import androidx.appcompat.app.AppCompatActivity
 
+
 class MainActivity : AppCompatActivity() {
+
+    val startForResult = registerForActivityResult(ActivityResultContracts.StartActivityForResult())
+    {
+        if (it.resultCode == Activity.RESULT_OK) {
+            //  you will get result here in result.data
+            val string:String = it?.data?.extras?.get("msgBack").toString()
```
Comment (hungvv-1301) [inline app/src/main/java/com/sun/android/MainActivity.kt]: Một số string như msgBack kia nên để biến hằng số bên ngoài . Mình sẽ dùng ở 2 màn mà . Nó sẽ thường có dạng như
EXTRA_MESSAGE..

File: app/src/main/java/com/sun/android/MainActivity.kt [hunk 2]
```diff
@@ -1,11 +1,43 @@
...
+            val txtRev: TextView = findViewById(R.id.text_message_reply)
+            val txtHead:TextView = findViewById(R.id.text_header_reply)
```
Comment (minhnn-1279) [inline app/src/main/java/com/sun/android/MainActivity.kt]: bài tập sau đổi sang sài viewBinding nhé :v

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -7,12 +7,54 @@
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/text_header_reply"
```
Comment (hungvv-1301) [inline app/src/main/res/layout/activity_main.xml:10]:
```suggestion
        android:id="@+id/text_view_header_reply"
```

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -7,12 +7,54 @@
...
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
```
Comment (hungvv-1301) [inline app/src/main/res/layout/activity_main.xml:12]: tools:text = ""

**Review by hungvv-1301 — APPROVED** [review]

File: app/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -5,12 +5,20 @@
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
+        android:label="Two Activities"
```
Comment (ghost) [inline app/src/main/AndroidManifest.xml:8]: dùng string resource

File: app/src/main/java/com/sun/android/newA.kt [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+class newA : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_new)
+        val actionBar = getSupportActionBar()
+        actionBar?.setDisplayHomeAsUpEnabled(true)
+        val receivedMsg = findViewById<TextView>(R.id.text_message)
+        val value:Bundle = intent.extras!!
```
Comment (ghost) [inline app/src/main/java/com/sun/android/newA.kt]: không dùng assert non null

**Pull request merged**